index.js

```diff
@@ -1,16 +1,35 @@
 'use strict';
 
-const exec = require('child-process-promise').exec;
+const fs = require('fs');
+const fsp = require('fs-promise');
 const findUp = require('find-up');
 const ora = require('ora');
+const exec = require('child-process-promise').exec;
 
 const CONFIG_NAME = 'yolo.json';
 
-module.exports = function run() {
+function initialize() {
+  const spinner = ora().start();
+  const configPath = `./${CONFIG_NAME}`;
+  const examplePath = `${__dirname}/${CONFIG_NAME}.example`;
+
+  spinner.text = 'Checking if file exists...';
```

Comment:
I would move lines 16-20 and 22-26 to separate methods because of different functionalities